README.md

```diff
@@ -100,35 +100,35 @@ and then uses it to install several modules:
+                                "instance_type": "t2.micro",
+                                      "region": "us-east-1",
+                                            "type": "amazon-ebs",
+                                                  "source_ami": "ami-fce3c696",
+                                                        "ssh_username": "ubuntu"
+                                                            }
+                                                              ],
+                                                                "provisioners": [
+                                                                    {
+                                                                            "type": "shell",
+                                                                                  "inline":
+                                                                                          "curl -Ls https://raw.githubusercontent.com/gruntwork-io/gruntwork-installer/v0.0.38/bootstrap-gruntwork-installer.sh | bash /dev/stdin --version v0.0.16"
+                                                                                              },
+                                                                                                  {
+                                                                                                          "type": "shell",
+                                                                                                                "inline": [
+                                                                                                                        "gruntwork-install --module-name 'ecs-scripts' --repo 'https://github.com/gruntwork-io/terraform-aws-ecs' --tag 'v0.0.1'",
+                                                                                                                                "gruntwork-install --module-name 'fail2ban' --repo 'https://github.com/gruntwork-io/terraform-aws-security' -module-param 'ban-time=3600'",
+                                                                                                                                        "gruntwork-install --binary-name 'gruntkms' --repo 'https://github.com/gruntwork-io/gruntkms' --tag 'v0.0.1'"
+                                                                                                                                              ],
+                                                                                                                                                    "environment_vars": ["GITHUB_OAUTH_TOKEN={{user `github_auth_token`}}"]
+                                                                                                                                                        }
+                                                                                                                                                          ]
```

Comment:
I think this indentation was accidentally introduced, since it doesn't render correctly.

Reply:
Bleh! What the hell happened here...